.github/ISSUE_TEMPLATE/1-bug.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: bug
+labels: bug,needs-triage
```

Comment:
should we also remove https://github.com/openstreetmap/id-tagging-schema/labels/bug until someone has assesed if it's genuinely a bug?